### PR TITLE
Add WebGPU whisper large and llama 3.2

### PR DESCRIPTION
**Pull request by gyagp**

### CODE REVIEW

**Comment by gyagp** [comment]
@NingW101 @ibelem PTAL.

**Review by ibelem — APPROVED** [review]
LGTM, thanks @gyagp !

@NingW101

File: js/main.js [hunk 1]
```diff
@@ -466,43 +490,40 @@ function constructSampleHTML(samples) {
...
+
+                  ${sample.devices.length > 0
+          ? `${sample.devices
+            .map(
+              (device) =>
+                `<div class="flex 2xl:font-medium rounded-2xl bg-sky-600/80 px-2 text-stone-50">${device}</div>`
+            )
+            .join("")}`
+          : ``
+        }
```
Comment (NingW101) [inline js/main.js:526]: Could you please run `npm run format` in project root to update these formatting changes.

Comment (gyagp) [reply js/main.js]: Updated. However, I saw some format changes not caused by my PR, so I'm not sure if this is what you want.

**Comment by NingW101** [comment]
LGTM~ @ibelem Could you please help to merge this patch, thank

**Review by ibelem — APPROVED** [review]
LGTM, thanks @gyagp!

**Pull request merged**